_includes/v21.2/sidebar-data/get-started.json

```diff
@@ -22,26 +22,26 @@
           "title": "Hello World",
           "items": [
             {
-              "title": "Go",
+              "title": "Build a Hello World App Using Go",
```

Comment:
Comment got lost somehow. Or I probably forgot to add it. 

I'm not sure why we're spelling this out in each case. It makes the sidenav very cluttered:

![Screen Shot 2022-01-31 at 12 03 04 PM](https://user-images.githubusercontent.com/15893490/151855837-fc3bbdec-7478-4dc1-830e-02da3ae1b8c7.png)

Perhaps we're trying to ensure that the sidenav link matches the page title, but that's also not true in these cases.